styles/HomepageSubnav.module.scss

```diff
@@ -0,0 +1,113 @@
+.container {
+    margin-top: $grid * 2;
+}
+
+.navigation {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: $grid;
+
+    @include media-query(medium) {
+        margin-bottom: $grid * 2;
+    }
+
+    @include media-query(large) {
+        margin-bottom: $grid * 2.5;
+    }
+}
+
+.navItem {
+    margin-right: $grid;
+
+    @include media-query(medium) {
+        margin-right: $grid * 2;
+    }
+}
+
+.jobNavItem {
+    position: relative;
+
+    @include media-query(medium) {
+        margin-left: $grid * 1.5;
+
+        &::before {
+            content: '';
+            position: absolute;
+            top: -5px;
+            left: -33px;
+            width: 1px;
+            height: 39px;
+            background: #000000;
```

Comment:
You can usually shorten hex colours like this to 3 digits i.e. `#000`

**Chore:** replace with `--color--black` variable.